pages/episodes/[episode].js

```diff
@@ -0,0 +1,29 @@
+import EpisodeDetails from "@/components/EpisodeDetails";
+import { useRouter } from "next/router";
+import useSWR from "swr";
+
+// this will change when using remote API
+const URL = "/serie.json";
+
+export default function EpisodeDetailPage() {
```

Comment:
I think `EpisodeDetailsPage` with a "s" would be better. 🤔

Reply:
changed it